### PR TITLE
Merge: Check builds

### PR DESCRIPTION
**Pull request by Chemin1**
This PR is based on commit f09f302 from #3106 => Only the last 2 commits are relevant. It reenables all builds but disables the tests that fail because of LibGit2 memory leaks.

I plan to rebase and merge it after #3106 is merged. Alternatively, I could also merge it into #3106 now as its checks are not successful anyway atm.

### CODE REVIEW

**Comment by markus2330** [comment]
Thank you for creating the PR!

**Comment by Chemin1** [comment]
jenkins build libelektra please

**Comment by Chemin1** [comment]
jenkins build libelektra please

**Comment by Chemin1** [comment]
jenkins build libelektra please

**Comment by markus2330** [comment]
jenkins build libelektra please

**Comment by Chemin1** [comment]
jenkins build libelektra please

**Comment by markus2330** [comment]
jenkins build libelektra please

**Comment by Chemin1** [comment]
jenkins build libelektra please

**Comment by Chemin1** [comment]
jenkins build libelektra please

**Comment by markus2330** [comment]
@sanssecours thank you for the review!

**Comment by Chemin1** [comment]
jenkins build libelektra please

File: doc/help/kdb-cmerge.md [hunk 1]
```diff
@@ -27,6 +27,37 @@ On success the resulting keyset will be saved to mergepath.<br>
...
+Strategies offer fine grained control over conflict handling. The option is:
+
+- `-s <name>`, `--strategy <name>`: which is used to specify a strategy to use in case of a conflict
+
+## Strategies
+
+The available strategies are:
+
+- `abort`: the merge will abort if any conflict happens and merge the 3 key sets together otherwise.
+- `our`: This option forces conflicting keys to be auto-resolved cleanly by favoring `our`. Changes from the other key sets that do not conflict with the `our` version are reflected to the merge result. This works like the recursive strategy with the `ours` option from git-merge.
```
[the remark this answers is not in the document]
Comment (Chemin1) [reply doc/help/kdb-cmerge.md]: It's "reflected to" in the [git-merge documentation](https://git-scm.com/docs/git-merge), but I think you're right nonetheless.

File: scripts/admin/install-config-file [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# use the file name (without path) of <config file> as special path
+# in Elektra
+specialPathEnd=$(basename $their)
+if [ -z "$specialPathEnd" ]; then
+	specialPathEnd=$their
+fi
+base="${specialPathStart}${specialPathEnd}"
+
+# check if something is in <elektra path>
+if [ $(kdb ls $our | wc -c) = 0 ]; then
```
[the remark this answers is not in the document]
Comment (Chemin1) [reply scripts/admin/install-config-file]: You are right. This is already implemented in 38625899c7556b0120c88938604475ffd63ffbd5 from #3106. Sorry for the inconvenience!

**Comment by Chemin1** [comment]
@sanssecours Thank you so much for taking a look at this!

I'll take a closer look at the `printf` soon. I pushed all the other corrections in 2c6e270db79eb496de9f07b0b28762beb4ea74d6 in #3106 because 1. I'm so happy that this PR got through the build process and 2. to keep it on topic. #3106 in contrast is pretty chaotic already.

File: doc/help/kdb-cmerge.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (markus2330) [reply doc/help/kdb-cmerge.md]: They are maybe happy if we report them this.

[the remark this answers is not in the document]
Comment (Chemin1) [reply doc/help/kdb-cmerge.md]: I sent [a patch](https://public-inbox.org/git/CAFOfB_LMQBOFK5PzhezcMGshmETY0agVK_iRW7vEayR9=Oyaow@mail.gmail.com/T/#u) to their mailing list.

[the remark this answers is not in the document]
Comment (markus2330) [reply doc/help/kdb-cmerge.md]: Btw. never miss an opportunity to mention Elektra. E.g. in this case that you were working on a 3-way merge for Elektra using libgit2 :wink: Btw. I am interested how linuxtag was (next meeting).

[the remark this answers is not in the document]
Comment (Chemin1) [reply doc/help/kdb-cmerge.md]: The patch is [in cue](https://public-inbox.org/git/xmqq5zjrjhjy.fsf@gitster-ct.c.googlers.com/) :)

**Comment by Chemin1** [comment]
Merged into #3209

**Pull request closed without merging**